samples/python_interop/generating_n_random_bits/README.md

```diff
@@ -0,0 +1,5 @@
+# A simple Q# program to generate an array of N random bits and Python program to count the number of 1s returned in the result array
+
+## Description
```

Comment:
You'll need to update description to match the code change, and we don't need subtitle here